tasks-by-users/Naufil-Shemle/03-css-positioning/style.css

```diff
@@ -0,0 +1,148 @@
+body {
+  background: #c2c2c2;
+}
+.wrapper {
+  width: 960px;
+
+  margin: 0 auto;
+  padding: 20px;
+
+  background: #fff;
+}
+
+.wrapper:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.services {
+  max-height: 150px;
+  overflow: hidden;
+}
+
+.service h2,
+.service p {
+  color: #999;
+  margin: 10px auto;
+  text-align: center;
+}
+
+.service {
+  float: left;
+  position: relative;
+
+  background: #eee;
+
+  width: 29.33%;
+  height: 140px;
+
+  margin: 1%;
+  padding: 1%;
+}
+
+.service:nth-child(1) {
+  top: 5px;
+}
+
+.service:nth-child(2) {
+  top: 15px;
+}
+
+.service:nth-child(3) {
+  top: 25px;
+}
+
+.services:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.content {
+  padding: 10px;
+  clear: left;
+  /* If we just use this "clear" and then we give the large "margin-top" value so it will take margin from "wrapper" not from the "services". If we want it should take margin from the "services" so we need to use the "services:after" pseudo class  */
+  margin: 20px 1%;
+}
+
+.content p {
+  column-count: 3;
+  column-width: 50px;
+  column-rule: 1px solid #ccc;
+
+  text-align: justify;
+}
+
+section,
+aside {
+  float: left;
+
+  width: 46%;
+
+  margin: 1%;
+  padding: 1%;
+
+  background: #f2f2f2;
+}
+
+aside img {
+  width: 100%;
+}
+
+.columns:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+#banner {
+  position: relative;
+  margin: 1%;
+  max-height: 300px;
+  overflow: hidden;
+}
+
+#banner img {
+  width: 100%;
+}
+
+#banner h2 {
+  position: absolute;
+  top: 30px;
+
+  width: 100%;
+
+  margin: 0;
+
+  text-align: center;
+
+  color: #fff;
+}
+
+nav {
+  position: fixed;
+  left: 0;
+  top: 0;
+
+  width: 100%;
+  background: #333;
+  z-index: 1;
+}
+
+nav ul:after {
+  content: "";
+  display: block;
+  clear: both;
```

Comment:
Using these properties 3 times above, use it in once using multiple selector `,`

Reply:
I will keep this point from next time.

Reply:
At this initial stage I kept this separately